### DISCUSSION TITLE
The count of 'from(Service.*).count();' and 'from(ServiceInstance.*).count()' are inconsistent

### DISCUSSION THREAD

**Discussion by pengyongqiang666** [apache/skywalking#11216] [Q&A]
I have added custom scripts in the OAL file: 'service_count = from(Service.).count();' and 'instance_count = from(ServiceInstance.).count()'. Some Services are experiencing issues。e.g: There is a Service with two instances, and the instance_count for these instances is inconsistent with the service_count.

**Comment by wu-sheng** [comment]
Most likely, your cluster mode is not set correctly if your OAPs are running more than one node.

**Reply by pengyongqiang666** [reply]
I'm running a single node oap

**Reply by pengyongqiang666** [reply]
In theory these two values should be the same, right

**Reply by wu-sheng** [reply]
Yes, and we are consistent.

**Reply by pengyongqiang666** [reply]
If there are fluctuations in the ES network causing partial persistence data failures, would this situation exist?

**Reply by wu-sheng** [reply]
Yes, it could. Such as one bulk of metric flush was timeout, then the value could be inconsistent.
